mailbox/event/event-rabbitmq/pom.xml

```diff
@@ -117,4 +117,19 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>true</reuseForks>
+                    <!-- Fail tests longer than 1 hours, prevent form random locking tests -->
+                    <forkedProcessTimeoutInSeconds>3600</forkedProcessTimeoutInSeconds>
```

Comment:
With recent code changes this override to specify a non default `forkedProcessTimeoutInSeconds` can be removed.